docs/aws/README.md

```diff
@@ -3,31 +3,35 @@
 This guide will provide a complete Karpenter installation for AWS.
 These steps are opinionated and may need to be adapted for your use case.
 
+>Note: This guide should take less than 1 hour to complete and should cost less than $.25
+
 ## Environment
 ```bash
-CLOUD_PROVIDER=aws
+export CLUSTER_NAME=$USER-karpenter-demo
+export AWS_DEFAULT_REGION=us-west-2
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-CLUSTER_NAME=$USER-karpenter-demo
-AWS_DEFAULT_REGION=us-west-2
+LATEST_KARPENTER_VERSION=$(curl -fsSL \
+  https://api.github.com/repos/awslabs/karpenter/releases/latest \
+  | jq -r '.tag_name')
 ```
 
 ### Create a Cluster
 
-Create an EKS cluster
+The example EKS configuration uses [AWS Fargate](https://aws.amazon.com/fargate/) to run the required Karpenter components, but it is not a requirement if you have your own clusters.
```

Comment:
Karpenter can run anywhere, including on self-managed node groups, managed node groups (link), or aws fargate (link). This demo will run Karpenter on Fargate, which means all EC2 instances added to this cluster will be controlled by Karpenter autoscaling.